tests/test_cirq.py

```diff
@@ -59,9 +59,9 @@ def assert_gates_equivalent(
     assert c.depth == target_depth
     if accelerators and not backend.supports_multigpu:
         with pytest.raises(NotImplementedError):
-            final_state = backend.execute_circuit(c, np.copy(initial_state))
+            final_state = backend.execute_circuit(c, np.copy(initial_state))._state
```

Comment:
This should be replaced with
```suggestion
            final_state = backend.execute_circuit(c, np.copy(initial_state)).state()
```
to avoid using the "private" attribute. Same for three other instances in this file.